views/home.erb

```diff
@@ -3,19 +3,27 @@
   <head>
     <meta charset='UTF-8' />
     <title>Hangman</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link href="https://fonts.googleapis.com/css?family=Comfortaa|Inconsolata" rel="stylesheet">
   </head>
-  <body class='container'>
-    <h1> WELCOME TO HANGMAN GAME</h1>
-    <img src='<%= @image %>.jpg' alt='hangman image' height="500" width="500">
-    <p><%= @secret_word %></p>
-    <form action='/play' method='post'>
-      <label for='letter'>Make a guess:</label>
-      <input type='text' id='letter' name='letter_chosen' autofocus="autofocus">
-      <button type='submit' class='btn btn-primary'>submit</button>
-    </form>
-    <p><%= @letters %></p>
-    <p><%= @show_word %></p>
-    <a href='/games-won'>How many times did I win?</a>
+  <body>
+    <div class='container'>
+      <div class='link'>
```

Comment:
I think the indentation is not quite right. Looks like there are two spaces here (which I think is usually what you do with HTML) but then below there is only one space.